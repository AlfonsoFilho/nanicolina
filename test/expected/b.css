.by {
  display: none !important;
}
.bz {
  display: block !important;
}
.ca {
  visibility: hidden;
}
.cb {
  display: none !important;
  visibility: hidden !important;
}
.cc,
.cd,
.ce,
.cf {
  display: none !important;
}
