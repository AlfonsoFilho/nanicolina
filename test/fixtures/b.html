<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Test B</title>
</head>
<body>
  <div class="container-fluid">
    <div class="row">
      <div class="col-xs-1 col-sm-8"></div>
      <div class="col-xs-1 col-sm-8"></div>
    </div>
    <div class="row">
      <div class="col-xs-12 col-lg-6"></div>
    </div>
  </div>
</body>
</html>
