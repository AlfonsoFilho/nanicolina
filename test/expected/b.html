<!DOCTYPE html>
<html lang="en">
<head>
  <meta charset="UTF-8">
  <title>Test B</title>
</head>
<body>
  <div class="a">
    <div class="b">
      <div class="bf c"></div>
      <div class="bf c"></div>
    </div>
    <div class="b">
      <div class="bu z"></div>
    </div>
  </div>
</body>
</html>
